lib/app/services/api.dart

```diff
@@ -1,8 +1,12 @@
 import 'dart:convert';
 
 import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
 
 import 'package:schulcloud/core/data.dart';
+import 'package:schulcloud/courses/data/content.dart';
```

Comment:
It may be useful to add a `data.dart` file to the `courses` module that simply exports all the data files to avoid import cascades like this.